modules/core/src/main/java/org/apache/ignite/internal/processors/platform/services/PlatformServices.java

```diff
@@ -822,5 +826,7 @@ private static void writeFailedConfiguration(BinaryRawWriterEx w, ServiceConfigu
         if (svcCfg.getNodeFilter() instanceof PlatformClusterNodeFilterImpl)
             dotnetFilter = ((PlatformClusterNodeFilterImpl)svcCfg.getNodeFilter()).getInternalPredicate();
         w.writeObjectDetached(dotnetFilter);
+
+        w.writeBoolean(svcCfg.isStatisticsEnabled());
```

Comment:
Should we write known methods here?

Reply:
Back to .NET? Not sure. The platform doesn't need to know the methods were tried to get covered by the statistics. On the platform side the methods are known.